islandora_book.module

```diff
@@ -371,3 +371,15 @@ function islandora_book_islandora_paged_content_content_model_registry() {
     ],
   ];
 }
+
+/**
+ * Implements hook_islandora_solution_pack_child_relationships().
+ */
+function islandora_book_islandora_solution_pack_child_relationships($cmodels) {
+  if ($cmodels === 'all' || in_array('islandora:bookCModel', $cmodels)) {
+    return [
+      'prefix' => ['PREFIX islandora: <http://islandora.ca/ontology/relsext#>'],
+      'predicate' => ['<islandora:isPageOf>'],
```

Comment:
Looks like [they're there upstream](https://github.com/Islandora/islandora_solution_pack_book/blob/b88210573ad75e2cd7d59b5d4da4d0a696427a7d/islandora_book.module#L445); but, the angle brackets aren't really necessary with the `PREFIX` being declared.